application/config/database.php.copy

```diff
@@ -76,9 +76,9 @@ $query_builder = TRUE;
 $db['default'] = array(
 	'dsn'	=> '',
 	'hostname' => 'localhost',
-	'username' => '',
+	'username' => 'root',
 	'password' => '',
-	'database' => '',
+	'database' => 'citemplate_db',
```

Comment:
You dont have to edit this file anymore. This is just for reference in creating database.php file